shell/assets/styles/base/_color.scss

```diff
@@ -10,9 +10,18 @@
   color: var(--disabled-text) !important;
 }
 
-$blue0001: #1F67DB;
+$blue001: #1F67DB;
```

Comment:
question: Is this the way that color variables are named in the design system doc? I think there is great benefit to be had in maintaining a a consistent naming convention between the design system and implementation.

@oboc-sts @edenhernandez-suse